src/lib/analytics/helpers.ts

```diff
@@ -4,7 +4,7 @@ import { extend, omit, pick } from "lodash"
 import { data as sd } from "sharify"
 import { trackTimeOnPage } from "./timeOnPageListener"
 import { setAnalyticsClientReferrerOptions } from "./setAnalyticsClientReferrerOptions"
-const setupSplitTests = require("../components/split_test/setup.coffee")
+const setupSplitTests = require("../../desktop/components/split_test/setup.coffee")
```

Comment:
Not a required change, however, we should note files going forward that reach out of `lib` as they have the potential to create dependency cycles.

Reply:
hopefully we can refactor the split test component and move it soon too!